Sanity-Vercel/README.md

```diff
@@ -25,8 +27,6 @@ Vercel CLI
 
 [Setup sanity studio](docs/sanity/setup.md)
 
-[Setup a nextjs project](docs/nextjs/setup.md)
-
 ## Excercises
```

Comment:
Should we add somewhere that a requirement for the exercises is also to make sure to have strong typing and avoid `any` as much as possible?

Reply:
Very good point @AlicinaCat , I added a comment about it in this file, let me know if you think it works, or if we should be more explicit

Reply:
I think it looks good now @elindanielsson :)